storybook/stories/wallets/send/WalletSend.stories.tsx

```diff
@@ -247,22 +318,23 @@ storiesOf('Wallets / Send', module)
       assets={sendFormAssetData}
       addressValidator={() => true}
       onSubmit={action('onSubmit')}
-      isDialogOpen={() => boolean('isDialogOpen', false)}
       isRestoreActive={boolean('isRestoreActive', false)}
       walletAmount={new BigNumber(123)}
       hwDeviceStatus={HwDeviceStatuses.VERIFYING_TRANSACTION}
       isHardwareWallet={boolean('isHardwareWallet', true)}
```

Comment:
Seems like @dmitrii-gaico spotted another issue with storybook knobs. 

Namely, depending on which scenario that has the `isHardwareWallet` knob you select first, that default value stays even when switching to diferent scenario. Possibly this is how it is designed to work, but in our case, for HW scenarios it is wrong.

Eg. if you open `Send - no assets` first, then `isHardwareWallet` is set to `false`. Then, if you navigate to `Send - Hardware wallet verifying transaction`, `false` remains set, and as a result you're not seeing hardware wallet specific messages we're meant to see here. I did a quick experiment and hardcoded `isHardwareWallet` as `true` and that seems to help, so I would recommend to remove the knob completely and replace it with static value for all HW scenarios on send screen (otherwise it's no longer a hardware wallet scenario, right?).

Reply:
Well, Netlify is just a static page hosting. Is this not happening on local?